ddtrace/internal/datadog/profiling/stack_v2/src/sampler.cpp

```diff
@@ -24,33 +24,29 @@ Sampler::sampling_thread()
             });
         });
 
-        // If we've been asked to stop, then stop
-        if (!is_profiling.load()) {
-            break;
-        }
-
         // Sleep for the remainder of the interval, get it atomically
```

Comment:
Would it be worth doing another check for changed sequence number right before the sleep?  Or is the sample itself fast enough relative to the sleep that this wouldn't help much?

Reply:
We can, it's a cheap check.  Fixed.